packages/nimble-components/.npmignore

```diff
@@ -2,6 +2,7 @@
 !/dist/*.*
 !/dist/esm/**
 /dist/esm/**/tests
+!/dist/esm/**/tests/**/*.pageobject.js
```

Comment:
Nope, the tests namespace is private, don't make stuff there public
The `testing` namespace is where public test utilities are exported.
Move it to `@ni/nimble-components/dist/esm/table/testing/table.pageobject`

Also add the `tests` and `testing` dirs to the restricted imports list in nimble-angular so folks don't add test dependencies to production code: 
https://github.com/ni/nimble/blob/main/angular-workspace/.eslintrc.js#L11